Base/usr/share/man/man1/syscall.md

```diff
@@ -0,0 +1,58 @@
+## Name
+
+syscall - test a system call
+
+## Synopsis
+
+`syscall [-o] [-l] [-h] entry [args; buf==BUFSIZ buffer]`
+
+## Description
+
+syscall(1) is an effort to port the plan9 syscall command to Serenity. It can invoke a system call with the given arguments from the command line.
```

Comment:
Inside a man page, don't reference its subject as `name(section)` (that's how you reference *other* subjects). Just `syscall` (or "The `syscall` utility") in this case would be fine.

Reply:
Honestly, the Plan 9 mention belongs somewhere at the end (like maybe in a "history" or "according to" section).  And it should read "Plan 9", not "plan9".